core/CMakeLists.txt

```diff
@@ -151,4 +151,31 @@ if(THEROCK_ENABLE_HIP_RUNTIME)
       libhiprtc-builtins.so
       libhiprtc.so
   )
+  if(WIN32)
+    ##############################################################################
+    # hipInfo (from hip-tests)
```

Comment:
We'll also want to update these files, though I'd leave that for a follow-up change:


* Add hipinfo launcher to python packages: https://github.com/ROCm/TheRock/blob/11739e9aed1c07d4b3ceba65c78dcaecb37eaf0c/build_tools/packaging/python/templates/rocm-sdk-core/src/rocm_sdk_core/_cli.py#L50-L63
* Add console script for that launcher: https://github.com/ROCm/TheRock/blob/11739e9aed1c07d4b3ceba65c78dcaecb37eaf0c/build_tools/packaging/python/templates/rocm-sdk-core/setup.py#L68-L79
* Add hipInfo to https://github.com/ROCm/TheRock/blob/main/docs/development/windows_support.md#supported-subprojects